sklearn/utils/__init__.py

```diff
@@ -57,7 +57,7 @@ class Parallel(_joblib.Parallel):
 
 __all__ = ["murmurhash3_32", "as_float_array",
            "assert_all_finite", "check_array",
-           "check_random_state",
+           "check_random_state", "check_psd_eigenvalues",
```

Comment:
Ok. I updated `classes.rst` for now because I'm more in favour of letting users play with it and possibly suggest to improve it further ; but we can surely wait for a third point of view here.

Reply:
(done - this is private)